docs/sources/developers/plugins/metadata.md

```diff
@@ -21,22 +21,28 @@ The plugin.json file is required for all plugins. When Grafana starts, it scans
 | `logs`          | boolean                 | No       | For data source plugins. If the plugin supports logs.                                                                                                                                                                                                                                                                                                                                                   |
 | `metrics`       | boolean                 | No       | For data source plugins. If the plugin supports metric queries. Used in the Explore feature.                                                                                                                                                                                                                                                                                                            |
 | `mixed`         | boolean                 | No       | Not to be used by external plugins. Special property for the built-in mixed plugin.                                                                                                                                                                                                                                                                                                                     |
 | `module`        | string                  | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
 | `partials`      | [object](#partials)     | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `preload`       | boolean                 | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
will make sure the `module.js` is executed on startup, not just when the plugin is first used.